appveyor.yml

```diff
@@ -1,4 +1,32 @@
-build: off
+image: Visual Studio 2017
+
+hosts:
+  example.letsencrypt.org: 127.0.0.1
+  elpmaxe.letsencrypt.org: 127.0.0.1
+
+environment:
+  GO111MODULE: on
+  GOPATH: C:\gopath
+
+install:
+  - set PATH=C:\Python37;C:\Python37\Scripts;%PATH%
+  - git clone https://github.com/certbot/certbot --branch fix-pipstrap-on-windows
+  - cd certbot
+  - python tools\venv3.py
+  - venv3\Scripts\activate.bat
+  - cd ..
+
+build_script:
+  - go install -v -mod=vendor ./...
+  - ps: $PebbleProcess = Start-Process C:\gopath\bin\pebble.exe -PassThru
 
 test_script:
-  - ps: Write-Host "Hello, world!"
+  - go vet ./...
```

Comment:
```suggestion
  - go vet -mod=vendor ./...
```

Reply:
For my curiosity, what means this option, as its effect to the build is not obvious to me?